docs/source/API/core/view/view.rst

```diff
@@ -68,35 +67,35 @@ and ``MemoryTraits`` are specified, ``MemorySpace`` must come before ``MemoryTra
 
 * ``MemorySpace``:
 
-  Controls the storage location.
-  If omitted the default memory space of the default execution space is used (i.e. ``Kokkos::DefaultExecutionSpace::memory_space``)
+  Views store data in Memory Spaces.
+  If omitted as a template parameter, the default memory space of the default execution space is used (*i.e.* ``Kokkos::DefaultExecutionSpace::memory_space``).
 
 .. _Atomic: ../atomics.html
 
 .. |Atomic| replace:: :cppkokkos:func:`Atomic`
 
 
 * ``MemoryTraits``:
-
-  Sets access properties via enum parameters for the templated ``Kokkos::MemoryTraits<>`` class.
+  MemoryTraits specify how a data structure is accessed.
+  Sets access properties via enum parameters for the templated ``Kokkos::MemoryTraits<>`` class.  
   Possible template parameters are bit-combinations of the following flags:
 
   - ``Unmanaged``: The View will not be reference counted. The allocation has to be provided to the constructor.
 
-  - |Atomic|_: All accesses to the view will use atomic operations.
+  - |Atomic|_: All accesses to the View will use atomic operations.
```

Comment:
if we capitalize we should make it code font. Note that "view" is a general word referring to what `View` is a: a data structure with "view" semantics.